### DISCUSSION TITLE
Aggregate harvester datas on the farmer

### DISCUSSION THREAD

**Discussion by hacknisty** [Chia-Network/chia-blockchain#5294] [Ideas]
Hi,

Sorry if this is a duplicate, i searched in the issues but couldn't find anything about this feature.

It would be great, on the farmer server, to display aggregated datas from others harvester. Currently only the local harvester is displayed.

Having :
 - Plot counts
 - Total size of plots
 - Expected time to win
 aggregated would be awesome.

Regards,

**Comment by lvcivs** [comment]
This has been suggested before and I believe that it is being worked on. Likely to be in one of the next releases.
